great-idea-website/css/index.css

```diff
@@ -63,4 +63,118 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+  width: 80%;
+  min-width: 600px;
```

Comment:
```suggestion
  min-width: 800px;
```

Reply:
I know I was the one who did 600, but it seems we're having weird behavior when the page is getting too small.